lib/watcher.js

```diff
@@ -44,12 +44,10 @@ function Watcher(logger, api, files, sources) {
 
 	this.isTest = makeTestMatcher(files, AvaFiles.defaultExcludePatterns());
 
-	var isFirstRun = true;
 	this.clearLogOnNextRun = true;
+	this.runVector = 0;
 	this.run = function (specificFiles) {
-		if (isFirstRun) {
-			isFirstRun = false;
-		} else {
+		if (this.runVector > 0) {
```

Comment:
Of those two I'd prefer `!== 0`, but I'm not sure why `> 0` is unacceptable. Because it implies `runVector` might be less than zero? Or some other reason?